parm/config/config.resources.nco.static

```diff
@@ -152,17 +154,17 @@ elif [ $step = "analdiag" ]; then
     export nth_analdiag=1
     export npe_node_analdiag=$npe_node_max
     if [[ "$machine" == "WCOSS_C" ]]; then export memory_analdiag="3072M"; fi
+    if [[ "$machine" = "WCOSS2" ]]; then export memory_analdiag="500GB"; fi
```

Comment:
Same question as above.
Seeing this is a single node job, may be request max memory instead of something that is arbitrarily smaller than the max.